### PR TITLE
modal click overlay to close

### LINKED ISSUE #37: add modal overlay click functionality to close

### PR DESCRIPTION
**Pull request by Syncretik**
resolves #37

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -99,6 +99,23 @@ export default function App({ Component, pageProps }: AppProps) {
   const dialogRef = useRef<HTMLDialogElement>(null);
   const contentRef = useRef<HTMLDivElement>(null);
 
+  const dialogClickHandler = (e: React.MouseEvent) => {
```
Comment (Riglanto) [inline pages/_app.tsx:102]: Does it work on mobile too? :grin:

**Review by Riglanto — APPROVED** [review]

File: pages/_app.tsx (hunk 1 above)
Comment (Riglanto) [reply pages/_app.tsx:102]: checked the preview on mobile, looks fine

**Comment by har777** [comment]
@Syncretik would this be doing something similar? : https://github.com/Glo-Foundation/glo-wallet/blob/6839c962d6f7ff21b6bc0c5e55e2da99296ff174/lib/hooks.ts#L3

**Comment by Syncretik** [comment]
> @Syncretik would this be doing something similar? :
> 
> https://github.com/Glo-Foundation/glo-wallet/blob/6839c962d6f7ff21b6bc0c5e55e2da99296ff174/lib/hooks.ts#L3

potentially but we did have something similar before with a nested div `contentRef` and listeners and that wasn't working :/

**Pull request merged**